### ISSUE TITLE
[Android][Accessibility] WCAG 1.4.3: Text elements must have sufficient contrast against the background. (android.widget.TextView)

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#4113]
#### Issue
`Text elements must have sufficient contrast against the background.` ([ColorContrast](https://accessibilityinsights.io/info-examples/android/color-contrast/))

![image](https://user-images.githubusercontent.com/4442788/83782313-a2b28800-a65d-11ea-92e2-21ce11482fa1.png)

Similarly w/ radio, checklist, and spinner inputs when using a dark host config (e.g. Teams Dark):
![image](https://user-images.githubusercontent.com/4442788/83783510-29fffb80-a65e-11ea-94c5-120a9514ec35.png)
![image](https://user-images.githubusercontent.com/4442788/83783926-4d2aab00-a65e-11ea-832d-1a0331a6f1f7.png)

#### Target application
io.adaptivecards.adaptivecardssample
#### Element path
android.widget.TextView
#### How to fix
    The text element has insufficient contrast of 2.553. Foreground color: #5e92c1, background color: #d7e4ef). Modify the text foreground and/or background colors to provide a contrast ratio of at least 4.5:1 for regular text, or 3:1 for large text (at least 18pt, or 14pt+bold).
#### Environment

This accessibility issue was found using Accessibility Insights for Android 2020.527.10 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
